### ISSUE TITLE
unexpected behavior when items are placed inside #container

### ISSUE THREAD

**Issue by kordeviant** [rintoj/angular2-virtual-scroll#153]
there is no documentation on the styling of the container element and when no style is applied, count of objects that get created is more than viewable count(different from when items are not inside #container)

**Comment by samsamm777** [comment]
@kordeviant im having the same issue. i cant work out what the css should be.

**Comment by speige** [comment]
I agree this doesn't seem to be working well. I'm not sure if I'm using it wrong, or if it's just busted. If anyone is able to get this working, please submit a, please submit a pull request. (or if I'm just using it wrong, submit an example so I can add it to the demo).
